src/panel/poi/blocks/Hour.jsx

```diff
@@ -16,16 +18,11 @@ export default class HourBlock extends React.Component {
       return null;
     }
 
-    return <div className="poi_panel__info__section poi_panel__info__section--hours">
-      <div className="poi_panel__info__section__description">
-        <div className="icon-icon_clock poi_panel__block__symbol"></div>
-        <div className="poi_panel__block__content">
-          <TimeTable
-            schedule={schedule}
-            title={this.props.covid19enabled && _('See the usual opening hours', 'covid19')}
-          />
-        </div>
-      </div>
-    </div>;
+    return <Block icon="icon_clock" title={_('Opening Hours')}>
```

Comment:
No need for a capital on "Opening **H**ours", and there'ss already an i18n string `'Opening hours'`.